rest-api-spec/src/main/resources/rest-api-spec/test/cat.indices/10_basic.yaml

```diff
@@ -71,6 +71,41 @@
                 $/
 
 ---
+"Test cat indices using health status":
+  - do:
+      indices.create:
+        index: foo
+        body:
+          settings:
+            number_of_shards: "1"
+            number_of_replicas: "0"
+  - do:
+      indices.create:
+        index: bar
+        body:
+          settings:
+            number_of_shards: "1"
+            number_of_replicas: "1"
```

Comment:
@javanna I pushed b0ad122d52aadbe2bd9530dfa0b92d8bd7043fe8, can you take a look?